tck/bridge-tck-main-portlet/src/main/webapp/tests/resourcesRenderedInHeadTest.xhtml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<f:view xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://xmlns.jcp.org/jsf/core"
+		xmlns:h="http://xmlns.jcp.org/jsf/html" xmlns:portlet="http://xmlns.jcp.org/portlet_3_0"
+		xmlns:c="http://java.sun.com/jsp/jstl/core">
+	<h:head>
+		<h:outputScript library="test" name="resourcesRenderedInHeadTest.js" />
+		<portlet:resourceURL var="testResource1">
+			<portlet:param name="ln" value="test" />
+			<portlet:param name="javax.faces.resource" value="resource1.js" />
+		</portlet:resourceURL>
+		<script type="text/javascript" src="${testResource1}"></script>
```

Comment:
@stiemannkj1 Should we test for a plain `<link>` resource in the same way we are testing for the plain `<script>` resource?